tools/backups2datalad/asyncer.py

```diff
@@ -215,6 +223,11 @@ async def process_asset(
                 self.log.debug("%s: Skipping asset", asset.path)
                 self.tracker.finish_asset(asset.path)
                 return
+            if self.error_on_change:
+                raise UnexpectedChangeError(
```

Comment:
not really clear to me why getting to this point means that we got it changed "unexpectedly". Could you please elaborate?  Might be worth stating that in the comment for future.

Reply:
I think I kinda got it while reviewing code more, and there are multiple places. So may be let's not bother on this one.